### DISCUSSION TITLE
Multi-vertex polylines or tracing paths

### DISCUSSION THREAD

**Discussion by themopoco** [opengisch/QField#5182] [Q&A]
Hi there, 
I cannot find a similar question in the forum yet.
I am new to QField, using Android and connecting to Emlid Reach RS2+

I am interested in creating lines following a path (that is not always straight), in other words, I want to track individual line features. I have tried with a geo package layer of a multiline and a poyline shapefile, but while in the field and start the new feature Qfield only creates straight lines (cannot trace all my small turns and paths). How can I trace paths as features (with their unique attributes)?

I have tried with tracking but that does not create individual features, but an aggregate of all my walking in that layer.
Is there something in preparing the project that I am doing wrong? or the settings in QField?
Any advice?

Thank you!

**Comment by VxTedxV** [comment]
Tracking is what you're searching for. Each time you stop tracking, it saves a feature. Start and stop tracking for each track you want to save.
https://docs.qfield.org/how-to/tracking/

**Reply by themopoco** [reply]
Thanks for the quick response @VxTedxV!
I guess Qfield is still a little finicky on tracking and I wasn't so intuitively to me.

Sometimes (can't figure out why) tracking starts from last place I stopped tracking on that layer even if I did manually stop. It happened to me the very few first times I experimented with tracking and then again other times. I closed the app and started again and the problem was fixed but is not ideal.

Thanks again.

**Reply by nirvn** [reply]
@themopoco , QField supports track resumption, make sure you click on the Start Tracking and not Resume Tracking.
